src/BloomExe/Program.cs

```diff
@@ -250,9 +250,17 @@ static void Main(string[] args1)
 #if DEBUG
 						StartDebugServer();
 #endif
-						L10NSharp.LocalizationManager.SetUILanguage(Settings.Default.UserInterfaceLanguage, false);
+						LocalizationManager.SetUILanguage(Settings.Default.UserInterfaceLanguage, false);
+
+						// BL-1258: sometimes the newly installed fonts are not available until after Bloom restarts
+						if (FontInstaller.InstallFont("AndikaNewBasic"))
+						{
+							var restartMsg = LocalizationManager.GetDynamicString("Bloom", "Startup.FontInstalled",
+								"New fonts have been installed. Bloom will shut down now, and when you restart it the fonts will be available.");
```

Comment:
I removed the message box because it was confusing after I got Bloom to restart automatically. No UI was visible yet, not even the splash screen, so now it just restarts without notifying the user.